packages/sandpack-core/src/npm/dynamic/fetch-protocols/index.ts

```diff
@@ -28,17 +27,23 @@ const protocols: ProtocolDefinition[] = [
     protocol: new JSDelivrGHFetcher(),
     condition: (name, version) => isGithubDependency(version),
   },
-  {
-    protocol: new ProtocolTransformer(new TarFetcher(), (name, version) => [
-      name,
-      version.replace(
-        'https://registry.npmjs.org/',
-        'https://registry.npmjs.cf/'
-      ),
-    ]),
-    condition: (name, version) =>
-      version.startsWith('https://registry.npmjs.org/'),
-  },
+  /**
+   * npmjs.cf has been deprecated, while registry.npmjs.org has CORS supports now:
+   *
+   * - npmjs.cf depreacation notice: https://github.com/npmjs-cf/meta/issues/8
+   * - npm registry CORS support: https://github.com/npm/feedback/discussions/117#discussioncomment-2691120
+   */
+  // {
+  //   protocol: new ProtocolTransformer(new TarFetcher(), (name, version) => [
```

Comment:
Shouldn't we still have the `TarFetcher` here? I think this protocol should add support for `https://registry.npmjs.org/'` dependencies.

Reply:
> Shouldn't we still have the `TarFetcher` here? I think this protocol should add support for `https://registry.npmjs.org/'` dependencies.

I though the `isTarDependency()` condition and the `TarFetcher` declared below have covered this: https://github.com/SukkaW/codesandbox-client/blob/2c06555eab9e69f7e889c524dbf674e997f0d251/packages/sandpack-core/src/npm/dynamic/fetch-protocols/index.ts#L48-L50

Reply:
@CompuIves

Reply:
@CompuIves Kindly request your attention with a friendly ping. It has been 2 weeks since your last review and my response. I would greatly appreciate it if you could spare some time to provide your valuable feedback at your convenience.

Reply:
@CompuIves Kindly request your attention with a friendly ping. It has been another 2 weeks. I would greatly appreciate it if you could spare some time to provide your valuable feedback at your convenience.